### DISCUSSION TITLE
Welcome to cryptofeed Discussions!

### DISCUSSION THREAD

**Discussion by bmoscon** [bmoscon/cryptofeed#351] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by yohplala** [comment]
Hi @bmoscon , hi everybody!
Glad about this terrific project, and glad you are open to contributions Bryant!
I am not an expert in coding, but I am an enthusiast in crypto and in ML stuff.
So it makes sense for me joining both :).
Who says ML naturally says gathering data first, and here I am in cryptofeed & cryptostore :)
Happy to share my contributions so that we can build together a versatile and robust tool!

**Comment by RezDaniel** [comment]
Nice!... Thank you @bmoscon and the other contributors to this amazing application. glad to be part of the discussions. Rez

**Comment by RezDaniel** [comment]
Hi @bmoscon,

What files do I need to change to collect trade data and (funding) from all BitMEX symbols, via Cryptofeed from your supplied docker build?

Many thanks

**Reply by bmoscon** [reply]
@RezDaniel - sorry, didnt see this - for questions like this its probably better to just open a new thread so I can more easily see it instead of asking the question on a thread thats not related. No worries though. Bitmex has a method that returns all trading pairs, you can subscribe to that to get all the data, just make sure that you have the trades channel configured.

**Comment by Physicworld** [comment]
Hey! i love your work, i really apreciate the people who develop free code to other coders.
Actually you save us a lot of coding time.
I would like yo know if the private websocket data for check the order states is gonna be able for binance futures and bybit?

**Reply by bmoscon** [reply]
at some point, yes. I rely a lot on contributors, so if one of them gets to it, it will be soon. If not, it may be a while, I have other things that are higher on my list
